datetimeparser/enums.py

```diff
@@ -1,41 +1,44 @@
+    EASTERN = Constant('eastern', ['easter'], time_value=eastern_calc)
+    NICHOLAS = Constant('nicholas', ['nicholas day'], time_value=lambda year_time: datetime(year=year_time, month=12, day=6))
+    HALLOWEEN = Constant('halloween', time_value=lambda year_time: datetime(year=year_time, month=10, day=31))
+    APRIL_FOOLS_DAY = Constant('april fools day', ['april fool day'], time_value=lambda year_time: datetime(year=year_time, month=4, day=1))
+    THANKSGIVING = Constant('thanksgiving', time_value=thanksgiving_calc)
+    SAINT_PATRICKS_DAY = Constant('saint patrick\'s day', ['saint patricks day', 'st. patrick\'s day', 'saint pt. day', 'st patrick\'s day', 'st patricks day'], time_value=lambda year_time: datetime(year=year_time, month=3, day=17))
+    VALENTINES_DAY = Constant('valentines day', ['valentine', 'valentine day'], time_value=lambda year_time: datetime(year=year_time, month=2, day=14))
+
+    SUMMER_BEGIN = Constant('summer begin', ['summer', 'begin of summer', 'begin of the summer'], time_value=lambda year_time: datetime(year=year_time, month=6, day=1))
+    WINTER_BEGIN = Constant('winter begin', ['winter', 'begin of winter', 'begin of the winter'], time_value=lambda year_time: datetime(year=year_time, month=12, day=1))
+    SPRING_BEGIN = Constant('spring begin', ['spring', 'begin of spring', 'begin of the spring'], time_value=lambda year_time: datetime(year=year_time, month=3, day=1))
     FALL_BEGIN = Constant('fall begin', ['fall', 'begin of fall', 'begin of the fall', 'autumn begin', 'autumn',
-                                         'begin of autumn', 'begin of the autumn'])
-    SUMMER_END = Constant('summer end', ['end of summer', 'end of the summer'])
-    WINTER_END = Constant('winter end', ['end of winter', 'end of the winter'])
-    SPRING_END = Constant('spring end', ['end of spring', 'end of the spring'])
-    FALL_END = Constant('fall end', ['end of fall', 'end of the fall', 'autumn end', 'end of autumn', 'end of the autumn'])
+                                         'begin of autumn', 'begin of the autumn'], time_value=lambda year_time: datetime(year=year_time, month=9, day=1))
+    SUMMER_END = Constant('summer end', ['end of summer', 'end of the summer'], time_value=lambda year_time: datetime(year=year_time, month=8, day=31, hour=23, minute=59, second=59))
+    WINTER_END = Constant('winter end', ['end of winter', 'end of the winter'], time_value=lambda year_time: datetime(year=year_time, month=2, day=days_feb(year_time), hour=23, minute=59, second=59))
+    SPRING_END = Constant('spring end', ['end of spring', 'end of the spring'], time_value=lambda year_time: datetime(year=year_time, month=5, day=31, hour=23, minute=59, second=59))
+    FALL_END = Constant('fall end', ['end of fall', 'end of the fall', 'autumn end', 'end of autumn', 'end of the autumn'], time_value=lambda year_time: datetime(year=year_time, month=11, day=30, hour=23, minute=59, second=59))
```

Comment:
Line too long, look at other lines where they've been already splitted onto multiple lines and do the same